modin/pandas/test/utils.py

```diff
@@ -630,8 +630,13 @@ def eval_general(
     comparator=df_equals,
     __inplace__=False,
     check_exception_type=True,
+    nonacceptable_exception_types=None,
     **kwargs,
 ):
+    if nonacceptable_exception_types not in (None, False):
```

Comment:
What types can this parameter be?

Can we not use bool values?

Reply:
`nonacceptable_exception_types ` can be `Exception` or list of `Exceptions`.
Ok, `not nonacceptable_exception_types` statement is used.